toqito/matrices/operator_sinkhorn.py

```diff
@@ -0,0 +1,198 @@
+"""Sinkhorn operator."""
+
+import warnings
+
+import numpy as np
+import scipy.linalg
```

Comment:
For consistency, let's `import scipy as sp` and then use `sp.linalg.sqrtm` when needed.